lib/model/budget.dart

```diff
@@ -9,60 +9,73 @@ class BudgetFields extends BaseEntityFields {
   static String name = 'name';
   static String idCategory = 'idCategory'; // FK
   static String amountLimit = 'amountLimit';
+  static String active = 'active';
   static String createdAt = BaseEntityFields.getCreatedAt;
   static String updatedAt = BaseEntityFields.getUpdatedAt;
 
   static final List<String> allFields = [
     BaseEntityFields.id,
     idCategory,
-    amountLimit,
     name,
+    amountLimit,
+    active,
     BaseEntityFields.createdAt,
     BaseEntityFields.updatedAt
   ];
 }
 
 class Budget extends BaseEntity {
   final int idCategory;
+  final String name;
   final num amountLimit;
-  final String? name;
+  final bool active;
 
   const Budget(
       {int? id,
       required this.idCategory,
+      required this.name,
       required this.amountLimit,
-      String? this.name,
```

Comment:
Why you added budget name as required field in this object? You also added this field in create table statement. It was removed on #71